src/python/grapl-tests-common/grapl_tests_common/clients/engagement_edge_client.py

```diff
@@ -1,35 +1,64 @@
+import hashlib
 import os
 from http import HTTPStatus
 from typing import Optional
 
+import boto3
 import requests
+from grapl_common.env_helpers import SecretsManagerClientFactory
+from grapl_common.grapl_logger import get_module_grapl_logger
+from grapl_tests_common.clients.common import endpoint_url
 
 _JSON_CONTENT_TYPE_HEADERS = {"Content-type": "application/json"}
 _ORIGIN = {
     "Origin": "https://local-grapl-engagement-ux-bucket.s3.amazonaws.com",
 }
 
+LOGGER = get_module_grapl_logger(default_log_level="DEBUG")
```

Comment:
not a blocker, but it's sort of odd for the default to be DEBUG

Reply:
I think it makes sense in test code, regardless I believe we're passing `GRAPL_LOG_LEVEL=DEBUG` into the environment (both in lambda and in the docker containers). If it becomes a problem can change it.